integration-tests/actor-tests/uniswap.test.ts

```diff
@@ -1,22 +1,16 @@
-import { BigNumber, Contract, utils, Wallet, ContractFactory } from 'ethers'
+import { Contract, utils, Wallet } from 'ethers'
 import { actor, run, setupActor, setupRun } from './lib/convenience'
 import { OptimismEnv } from '../test/shared/env'
-import { UniswapV3Deployer } from 'uniswap-v3-deploy-plugin/dist/deployer/UniswapV3Deployer'
-import { FeeAmount, TICK_SPACINGS } from '@uniswap/v3-sdk'
+import { FeeAmount } from '@uniswap/v3-sdk'
 import ERC20 from '../artifacts/contracts/ERC20.sol/ERC20.json'
+import { abi as NFTABI } from '@uniswap/v3-periphery/artifacts/contracts/NonfungiblePositionManager.sol/NonfungiblePositionManager.json'
```

Comment:
This is technically `NFPABI` or `NFPMABI` and not `NFTABI` being as its the non fungible position manager